package.json

```diff
@@ -131,6 +132,7 @@
 		"follow-redirects": "1.15.6",
 		"fs-extra": "11.1.1",
 		"hpagent": "1.2.0",
+		"node-fetch": "^2.7.0",
```

Comment:
This is not the latest version of node-fetch. I chose it because it already exists in our dependency tree.